src/Form/test-cases/apple_checkout.html

```diff
@@ -343,7 +343,7 @@ <h2 class="rs-address-subheader rf-form-layout-section-header">What’s your con
                                                             aria-labelledby="checkout.shipping.addressNotification.address.emailAddress_label"
                                                             aria-describedby="checkout.shipping.addressNotification.address.emailAddress_error "
                                                             aria-invalid="false" maxlength="50" aria-required="false"
-                                                            data-autom="form-field-emailAddress" value=""><span
+                                                            data-autom="form-field-emailAddress" value="" data-manual-scoring="emailAddress"><span
```

Comment:
From here on I'm just backfilling a bunch of tests that were missing manual scoring or where manual scoring was inaccurate. Feel free to take a look if you want, but now the automated tests report problems like typos and the likes so our coverage is overall much better.